### ISSUE TITLE
hybridloss

### ISSUE THREAD

**Issue by MinuteswithMetrics** [yingkaisha/keras-unet-collection#8]
is there plans to implement hybridloss?

**Comment by yingkaisha** [comment]
Do you have any example or paper handy that I can explore?

**Comment by aficionadoai** [comment]
Here's example [# 1](https://github.com/YifeiYang210/DSI-Brain-Tumor-Group10/blob/c742cf5c1f12278fe3ec23c43936a2ea21f24fe4/models/Unet3plus_tf2.py) and [ex2](https://github.com/ZJUGiveLab/UNet-Version/blob/master/loss/msssimLoss.py).  I believe the second example is from the paper.

**Comment by aficionadoai** [comment]
This may be a better [example ](https://github.com/Crispy13/Unet3plus_tensorflow2/blob/899e4c2aa8d064ee3bd6b9487c5549c061955728/Unet3plus/unet3plus.py)

**Comment by yingkaisha** [comment]
Looks like the functionality of hybrid loss is already covered by the Keras model API with `loss_weights`.

**Comment by yingkaisha** [comment]
@MinuteswithMetrics @aficionadoai I've added hybrid loss into the example notebook (`keras-unet-collection==0.1.6`).

https://github.com/yingkaisha/keras-unet-collection/blob/main/examples/segmentation_unet-three-plus_oxford-iiit.ipynb

MS-SSMI loss does not work well on categorical training labels, so it is not added into the hybrid.
